lemur/plugins/lemur_verisign/plugin.py

```diff
@@ -98,10 +98,16 @@ def process_options(options):
     :param options:
     :return: dict or valid verisign options
     """
+    # if there is a config variable with VERISIGN_PRODUCT_<upper(authority.name)> take the value as Cert product-type
+    # else default to "Server", to be compatoible with former versions
+    authority = options.get("authority").name.upper()
+    product_type = current_app.config.get("VERISIGN_PRODUCT_{0}".format(authority))
```

Comment:
One minor suggestion, to leverage  the default  option, and avoid the following if-statement.
``product_type = current_app.config.get("VERISIGN_PRODUCT_{0}".format(authority), "Server")``